phpunit.xml.dist

```diff
@@ -1,28 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
-<phpunit backupGlobals="false"
-         colors="true"
-         verbose="true"
-         bootstrap="tests/bootstrap.php"
-         failOnRisky="true"
-         failOnWarning="true">
-    <php>
-        <ini name="error_reporting" value="-1"/>
-    </php>
-
-	<testsuites>
-	    <testsuite name="Yii Db Pgsql Tests">
-			<directory>./tests</directory>
-		</testsuite>
-	</testsuites>
-
-    <filter>
-        <whitelist>
-            <directory>./</directory>
-            <exclude>
-                <directory>./tests</directory>
-                <directory>./vendor</directory>
-            </exclude>
-        </whitelist>
-    </filter>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" backupGlobals="false" colors="true" verbose="true" bootstrap="tests/bootstrap.php" failOnRisky="true" failOnWarning="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
```

Comment:
I'd not add XML namespace. It's not required and may cause Infection to fail.

Reply:
That is directly added by `phpunit with --migrate-configuration`, and it no longer affects infection.